app/services/git_hub_service.rb

```diff
@@ -0,0 +1,13 @@
+require 'httparty'
+
+class GitHubService
+  def self.get_contributors
```

Comment:
great work on formatting this GitHub service I will work on a test for this to test functionality!